Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -1327,8 +1387,7 @@ def main():
     verify_certificate = not params.get("insecure", False)
     proxy = params.get("proxy", False)
     hide_cvv_expiry = params.get("hide_data", False)
-    demisto.debug(f"Command being called is {params}")
-    mirror = params.get("mirror", False)
+    demisto.debug(f"params are: {params}")
```

Comment:
no need to add this log, when running a command with debug-mode=true, it will be printed without having to log this line
```suggestion
    command = demisto.command()
    demisto.debug(f"command being called is: {command}")
```